packages/lib/survey/service.ts

```diff
@@ -330,6 +331,73 @@ export const getSurveys = reactCache(
     )()
 );
 
+export const getSurveysForEnvironmentState = reactCache(
```

Comment:
same for this, please move this to `apps/web/app/api/v1/client/[environmentId]/environment/lib/survey.ts`